esi/src/config.rs

```diff
@@ -5,16 +5,20 @@
 /// let config = esi::Configuration::default()
 ///     .with_namespace("app");
 /// ```
+#[allow(clippy::return_self_not_must_use)]
 #[derive(Clone, Debug)]
 pub struct Configuration {
     /// The XML namespace to use when scanning for ESI tags. Defaults to `esi`.
     pub namespace: String,
+    /// Disables URL unescaping, only good with non-HTML content, eg JSON
+    pub is_escaped: bool,
```

Comment:
Such a nit-pick, sorry.
```suggestion
    /// Disables URL unescaping, only good with non-HTML content, e.g. JSON
```